EIPS/eip-1.md

```diff
@@ -187,7 +187,43 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
 
 ## Linking to External Resources
 
-Links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
+External resources are resources that are hosted outside of the EIPs repository. External resources may disappear, move, or change unexpectedly. The following rules apply in order.
+
+If you feel that a link you wish to include shouldn't violate the below rules, please submit a Pull Request to change them. Remember that the smaller and more specific the change, the more likely it will be accepted.
+
+### Access Rules
+
+- All external resources **MUST** be free to both access and distribute.
+- All external resources **MUST** use one of the following whitelisted URI schemes:
+  - `https`
+  - `ipfs`
+
+### Reference Rules
+
+- All external resources **MUST** be non-normative.
```

Comment:
-1 to this. One of the main reasons for addressing links in the first place would be to normatively reference other documents. Without this ability links still remain a blocker for 5593.

Reply:
Fair enough.